lib/msal-angular/docs/v2-docs/ssosilent.md

```diff
@@ -0,0 +1,54 @@
+# Silent login with ssoSilent()
+
+If you already have a session that exists with the authentication server, you can use the ssoSilent() API to make request for tokens without interaction. You will need to pass a `login_hint` (which can be retrieved from the account object `username` property or the `upn` claim in the ID token) in the request object in order to successfully obtain a token silently.
```

Comment:
```suggestion
If you already have a session that exists with the authentication server, you can use the ssoSilent() API to make request for tokens without interaction. You will need to pass a `loginHint` (which can be retrieved from the account object `username` property or the `upn` claim in the ID token) in the request object in order to successfully obtain a token silently.
```

Let's also callout that the `loginHint` can also come from the `emails` claim for B2C use cases and that you could alternatively use `sid` instead for AAD use cases.

Reply:
Good idea. Added.